lib/enmeshed/object.rb

```diff
@@ -4,20 +4,28 @@ module Enmeshed
   class Object
     delegate :klass, to: :class
 
-    def self.parse(content)
-      validate! content
-    end
+    class << self
+      def parse(content)
+        validate! content
+      end
 
-    def self.klass
-      name&.demodulize
-    end
+      def klass
+        name&.demodulize
+      end
 
-    def self.schema
-      @schema ||= Connector::API_SCHEMA.schema(klass)
-    end
+      def schema
+        @schema ||= Connector::API_SCHEMA.schema(klass)
+      end
+
+      def validate!(instance)
+        unless schema.valid?(instance)
+          error = schema.validate(instance).first.fetch('error')
+          Rails.logger.debug(error)
+          raise ConnectorError.new("Invalid #{klass} schema: #{error}") unless KNOWN_API_ISSUES.include? error
+        end
+      end
 
-    def self.validate!(instance)
-      raise ConnectorError.new("Invalid #{klass} schema") unless schema.valid?(instance)
+      KNOWN_API_ISSUES = ['value at `/auditLog/0/createdBy` does not match format: date-time'].freeze
```

Comment:
I can confirm this bug and created an issue: https://github.com/nmshd/feedback/issues/32

Reply:
My bug has been resolved and a new [connector version 6.13.2](https://github.com/nmshd/connector/releases/tag/6.13.2) has been released. I didn't test yet, but would hope that this new version actually fixes the error and removes the necessity for the `KNOWN_API_ISSUES`.

Reply:
I dropped the `KNOWN_API_ISSUES` and incorporated the other changes that came with the new connector API version

Reply:
Awesome, thanks!